PSICover/PSICover.cs

```diff
@@ -163,29 +165,75 @@ class Analyzer {
          var code = File.ReadAllLines (file);
          for (int i = 0; i < code.Length; i++)
             code[i] = code[i].Replace ('<', '\u00ab').Replace ('>', '\u00bb');
+         int covered = 0;

```

Comment:
Review from Arvind: Minor style guideline. I generally use lowercase c as a prefix to indicate a count. So this variable would be called `cHits`.